tests/end2end/features/utilcmds.feature

```diff
@@ -164,3 +164,7 @@ Feature: Miscellaneous utility commands exposed to the user.
     Scenario: Using debug-log-filter with invalid filter
         When I run :debug-log-filter blah
         Then the error "filters: Invalid value blah - expected one of: statusbar, *" should be shown
+
+    Scenario: Using debug-log-filter
+        When I run :debug-log-filter webview
+        Then no crash should happen
```

Comment:
I moved it out of the BDD tests and in to a regular pytest test. Seems to work well.

To reset the filter I had to add a special `none` argument.